src/components/Dashboard/index.jsx

```diff
@@ -0,0 +1,218 @@
+import { Component, Fragment } from 'react';
```

Comment:
Instead of:

![screen shot 2018-08-02 at 9 01 00 am](https://user-images.githubusercontent.com/3766511/43585237-a4d3af0c-9632-11e8-9269-133dd2b79839.png)

I was thinking something like:

![screen shot 2018-08-02 at 9 00 41 am](https://user-images.githubusercontent.com/3766511/43585261-b0588ffa-9632-11e8-8721-6ce37b0b0fb2.png)

What do you think?

Reply:
I'm open to a different design if you don't like the one I suggested.

Reply:
as for the padding-top in the skill drawer, i add it to make the 'Skills' word appear in the `/languages/:language` actually

Reply:
I don't think it's necessary to have the skills drawer when viewing the language view. We can remove it.